### ISSUE TITLE
Incorrect results in SBML semantic test case testId=998

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/AMICI#774]
Needs further investigations. Maybe just about error tolerances.
```
======================================================================
FAIL: test_sbml_testsuite (__main__.TestAmiciSBMLTestSuite) (testId=998)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./tests/testSBMLSuite.py", line 34, in test_sbml_testsuite
    self.run_sbml_testsuite_case(get_test_str(testId))
  File "./tests/testSBMLSuite.py", line 95, in run_sbml_testsuite_case
    np.isclose(simulated_x, test_x, atol, rtol).all()
AssertionError: False is not true
```

**Comment by paulstapor** [comment]
Unfortunately not a matter of tolerances: Amici Simulation fails for this model, since an `Inf` was encountered in the right hand side... Have to look into that more closely

**Comment by dweindl** [comment]
I shortly checked yesterday, it's about initial amount and initial concentrations. Check the description of the test case before diving too deep into something else.

**Comment by paulstapor** [comment]
yes, for some reason, it does not read the initial concentration correctly...
I don't really get it right now. But I will try to fix it tomorrow. Today ends now...

**Comment by dweindl** [comment]
Closed in #789
